### PR TITLE
BENCH-136: Update Order DTOs

### PR DESCRIPTION
**Pull request by mbaltrimas**

### CODE REVIEW

**Review by harrystead2 — APPROVED** [review]
Looks great! Thanks man

File: src/dtos/OrderDto.ts [hunk 1]
```diff
@@ -1,9 +1,16 @@
...
-  status: string;
-  address: string;
-  total: number;
-  items: OrderItemDto[];
-}
+import { LineItemDto } from './LineItemDto';
+
+export enum OrderStatus {
+  Created = 'Created',
+  Paid = 'Paid',
```
Comment (harrystead2) [inline src/dtos/OrderDto.ts:5]: Are we changing 'Paid' to 'Completed'? @milkshakeuk

**Review by kirbyAlasdairAnswer — APPROVED** [review]
Are we able to get some screenshots/gifs of the new functionality? or is the api not set up for these calls yet

**Comment by harrystead2** [comment]
> 

API not setup yet, need Callum's list change

**Pull request merged**